src/codemodder/codemods/api/helpers.py

```diff
@@ -47,18 +47,41 @@ def update_assign_rhs(self, updated_node: cst.Assign, rhs: str):
     def parse_expression(self, expression: str):
         return cst.parse_expression(expression)
 
-    def replace_arg(self, original_node, target_arg_name, target_arg_replacement_val):
-        """Given a node, return its args with one arg's value changed"""
+    def replace_arg(
+        self,
+        original_node,
+        target_arg_name,
+        target_arg_replacement_val,
+        add_if_missing=False,
+    ):
+        """Given a node, return its args with one arg's value changed.
+
+        If add_if_missing is True, then if target arg is not present, add it.
+        """
         assert hasattr(original_node, "args")
         new_args = []
+        arg_added = False
+
         for arg in original_node.args:
             if matchers.matches(arg.keyword, matchers.Name(target_arg_name)):
                 new = cst.Arg(
                     keyword=cst.parse_expression(target_arg_name),
                     value=cst.parse_expression(target_arg_replacement_val),
                     equal=arg.equal,
                 )
+                arg_added = True
             else:
                 new = arg
             new_args.append(new)
+
+        if add_if_missing and not arg_added:
+            new = cst.Arg(
```

Comment:
Just a small thing but it seems like this `Arg` creation could be factored out into another method.

Reply:
ah yes, thought about that and forgot, will do!